Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM ruby:2.3.0
+ARG CUSTOM_ASSETS_URL
```

Comment:
Overall, I would really prefer if we could do this without major modifications to the Dockerfile itself - at least, I'd love it if building didn't require three new arguments. As long as the assets are available to the rake task itself, it should suffice to issue `rake assets:precompile`.